crates/plugins/mediator-coordination/src/web/coord/handler/stateful.rs

```diff
@@ -337,16 +337,18 @@ pub async fn process_plain_keylist_update_message(
 
     let mediator_did = &state.diddoc.id;
 
-    Ok(Message::build(
-        format!("urn:uuid:{}", Uuid::new_v4()),
-        KEYLIST_UPDATE_RESPONSE_2_0.to_string(),
-        json!(KeylistUpdateResponseBody {
-            updated: confirmations
-        }),
+    Ok(
+        Message::build(
+            format!("urn:uuid:{}", Uuid::new_v4()),
+            KEYLIST_UPDATE_RESPONSE_2_0.to_string(),
+            json!(KeylistUpdateResponseBody {
+                updated: confirmations
+            }),
+        )
+        .to(sender.clone())
+        .from(mediator_did.clone())
```

Comment:
Why bother cloning these values here? They can be directly consumed.

Reply:
If the values `sender` and `mediator_did` are being reused after being passed into the `.to()` and `.from()` methods, I think it is necessary to clone them, but if not then it is unnecessary cloning as @Hermann-Core mentioned.